src/modules/invitation/invitation.controller.ts

```
@@ -0,0 +1,137 @@
+      acceptInvitationDto.token,
+      acceptInvitationDto,
+    );
+
+    const accessTokenExpirationMs = this.configService.jwtAccesTokenExpirationMs;
+    const refreshTokenExpirationMs = this.configService.jwtRefrehTokenExpirationMs;
+
+    // Définir les cookies pour les nouveaux tokens
+    res.cookie('access_token', accessToken, {
+      httpOnly: true,
+      secure: process.env.NODE_ENV === 'production',
+      maxAge: accessTokenExpirationMs ? parseInt(accessTokenExpirationMs, 10) : undefined,
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)